drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -291,5 +293,9 @@ def brief_name
     preferred_name || [first_name, last_name].compact.join(' ')
   end
 
+  def add_name(**attrs)
+    names.create!(data_source: data_source, user: user, **attrs)
+  end
+
```

Comment:
Was this just used for local testing? I think we can probably remove it since we likely won't be adding names this way. Still somewhat TBD but name creation/updating will happen either in the Client processor or in a ClientName processor.